samples/videoDecodeBasic/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+################################################################################
```

Comment:
Is this going to be a new sample? The `videoDecodeBasic` vs `videoDecode` names could be confusing

Reply:
This is a new sample. The reason for the "Basic" suffix is to tell it from the more general app videoDecode. Basic means it doesn't depend on FFMPEG libs.